DDDEastAnglia/Views/Account/_ExternalLoginMethodsPartial.cshtml

```diff
@@ -0,0 +1,12 @@
+﻿@model IEnumerable<DDDEastAnglia.Models.LoginMethodViewModel>
+
+@foreach (var externalLogin in Model)
+{
+    <li>
+        @using (Html.BeginForm("ExternalLogin", "Account", new { provider = externalLogin.ProviderName, returnUrl = Url.Action("Index", "Home") }, FormMethod.Post, new { id = externalLogin.ProviderName + "Loginform" }))
```

Comment:
Why is the `returnUrl` hard-coded here, rather than being set from the `ViewBag`?